plugins/plugins.go

```diff
@@ -34,48 +33,6 @@ const (
 	maxInterval     = 10 * time.Second
```

Comment:
nit: I think we are not using them. Can we remove?

Reply:
You're right..